### PR TITLE
Main content

### PR DESCRIPTION
**Pull request by thsvr**

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work @thsvr :clap: :clap: . However, some requirements that prevents your project's approval needs to be implemented. check below :arrow_down:  

#### Page structure

##### Navigation bar

* [x] It contains current date.
* [x] It contains main navigation links.
* [x] **It changes with RWD breakpoints.**

##### Main 

* [x] **It changes with RWD breakpoints.**
> In "featured Slideshows", something is causing an horizontal scroll with medium screen width

##### Footer

* [x] **It changes with RWD breakpoints.**
> The copyrights paragraph is also causing an horizontal scroll with small screen width

#### General changes:

* [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.

### That's it :relieved: . When finished, kindly submit another code review. Keep up the good work :muscle: .

**Comment by thsvr** [comment]
Hi!

All changes required by tse Ahmed were done. Can you please check if the project is able to be approved and then provide feedback? 

Thank you in advance :)

Cheers,

Thaís Vieira

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1367 @@
...
+    type="image/vnd.microsoft.icon" />
+</head>
+
+
+<header>
+  <!-- navbar fixed lg and xl screen-->
+  <nav class=" sticky-top d-none d-lg-flex flex-column d-sm-none p-2 mb-2">
+      <div id="top-nav" class="top-nav d-flex flex-row justify-content-between align-items-end pb-3 square-red">
+        <div id="date" class="date text-white pl-4">
+          <p>Wed, Sep 11, 2019</p>
```
Comment (samgaco) [inline index.html]: :+1:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1367 @@
...
+        </div>
+        <div id="top-logo" class="top-logo c-pointer">
+          <img src="assets/images/test-logo.jpg">
```
Comment (samgaco) [inline index.html]: `` tags should contain the alt attribute which gives a short description for the picture. More [here](https://validator.w3.org).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1367 @@
...
+  <!--subscribe newsletter-->
+  <div class="container mb-3 mt-4">
+    <div class="row">
+      <!--left content-->
+      <div class="col-lg-8 col-sm-12">
+        <div class="text-uppercase font-weight-bold red-color">Subscribe</div> <!--title-->
+        <div class="d-flex flex-row bg-light"> <!--to put content in a row-->
+          <div class="subs-left height-mg"> <!--magazines-->
+            <img class="pt-3 pl-3" src="assets/images/mg02.jpg" alt="images">
+            <img  class="mg-herb"src="assets/images/mg01.jpg" alt="images">
```
Comment (samgaco) [inline index.html]: You need spacing for the src attribute :smiley:

**Review by samgaco — CHANGES_REQUESTED** [review]
Hi @thsvr :hand: 

Everything has been implemented as the previous TSE requirements dictated :+1: Still there is one point to cover:

1. > Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.

**[REQUIERED]** You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.


---------------
Have these into consideration :arrow_down: 
* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

--------------

Summary: Address the errors in the validator and all the requested points will be covered :ok_hand: 

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

File: index.html (hunk 3 above)
Comment (thsvr) [reply index.html]: Thank you for the feedback, Samgaco  :) I did all the changes required and used the validator correctly.

Next time I will pay attention to ''meaningful names for the Pull Requests'' and will ''add a short summary'', so thanks for the advice.

**Comment by thsvr** [comment]
Hi!

All changes required by tse Samgaco were made in this second review - basically, add alt attribute in some . Can you please check if the project is able to be approved and then provide feedback?

Thank you :)

Regards

Thaís Vieira

**Review by Oghenebrume50 — APPROVED** [review]
Hi Team, everything looks good now I like that you guys made the corrections properly, please keep this up :smile:  
***This is approved***

**Pull request merged**